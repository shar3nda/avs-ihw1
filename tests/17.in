10
8 5 0 0 4 7 7 10 7 7