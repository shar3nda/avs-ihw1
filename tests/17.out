0 3 6 -1 2 4 0 10 8 6 6